.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: python
+python:
+- 2.7
+install:
+- pip install -r requirements.txt
+- pip install .
+- git clone https://github.com/napalm-automation/napalm-base.git napalm-automation/napalm-base
+- cd napalm-automation/napalm-base
+- python setup.py install
+- cd ../..
```

Comment:
Yes, I can take care about the deployment phase myself. Don't worry about
that.
On Mon 9 May 2016 at 14:48, Mircea Ulinic <notifications@github.com> wrote:

> In .travis.yml
> <https://github.com/napalm-automation/napalm-panos/pull/1#discussion_r62491921>
> :
>
> > @@ -0,0 +1,10 @@
> > +language: python
> > +python:
> > +- 2.7
> > +install:
> > +- pip install -r requirements.txt
> > +- pip install .
> > +- git clone https://github.com/napalm-automation/napalm-base.git napalm-automation/napalm-base
> > +- cd napalm-automation/napalm-base
> > +- python setup.py install
> > +- cd ../..
>
> Probably @dbarrosop <https://github.com/dbarrosop> will update thedeploy
> section later.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/napalm-automation/napalm-panos/pull/1/files/9836e79a0471a43cd38a298f0ad44f45448ec4d6..6908a1f0638373fbbb41452a30a036a648e20847#r62491921>
>